coa-helm/values.yaml

```diff
@@ -77,3 +88,9 @@ cbas:
   name: cbas
   image: us.gcr.io/broad-dsp-gcr-public/cbas:0.0.18
 
+wds:
+  name: wds
+  image: gcr.io/broad-dsde-dev/workspacedata

```

Comment:
Only that `cromwell-helm` doesn't. My goal was to copy `cromwell-helm`'s setup as closely as possible. We should continue to improve/unify both charts.

If we add a version here, we should add a version to `cromwell-helm`. We can decide collectively whether or not that belongs in this PR.

Reply:
ah, it seems that there is a reason: none of the `workspacedata` images are tagged in GCR (the best we can do is `latest`, which isn't that helpful)

Reply:
The _other_ reason is that we rely on this (slightly out of date) custom build, because the versioned images have breaking changes in them...

It's obviously not sustainable to keep doing it this way, we have a ticket to update the version the next time we touch WDS development